config/routes.rb

```diff
@@ -2,15 +2,19 @@
   root to: 'pages#front'
   get '/register', to: 'users#new'
   get '/login', to: 'sessions#new'
+  get '/logout', to: "sessions#destroy"
```

Comment:
Despite solution going for `GET` I would recommend using `DELETE` as that is more semantically correct. Also users will not be able to bookmark it.